# ATT
# https://github.com/diydrones/ardupilot/blob/master/libraries/DataFlash/DataFlash.h#L426
Header header

int16 DesRoll  # int16_t  control_roll;
int16 Roll     # int16_t  roll;
			   # int16_t  control_pitch; TODO:
int16 Pitch    # int16_t  pitch;
uint16 DesYaw  # uint16_t control_yaw;
uint16 Yaw     # uint16_t yaw;
uint16 ErrRP   # uint16_t error_rp;
uint16 ErrYaw  # uint16_t error_yaw;


    
    
    
    
    
    
    
    